README.md

```
@@ -6,6 +6,10 @@
 
   <img width="600" alt="image" src="https://github.com/user-attachments/assets/e120663e-0916-4c79-b0eb-b16034601b9c" />
 
+* In the forked repository settings (category **Actions**▶️**General**) activate the option `Allow GitHub Actions to create and approve pull requests` (s. [Workflow permissions](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#preventing-github-actions-from-creating-or-approving-pull-requests) in the GitHub documentation for details).
+
+  <img width="600" src="https://github.com/user-attachments/assets/b9977f37-e9b4-4a93-8dc2-24f592e2d973" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)